Makefile

```diff
@@ -311,7 +311,7 @@ test: test-unit test-acceptance
 build: out/operator
 
 out/operator:
-	$(Q)GOARCH=$(ARCH) GOOS=$(OS) go build ${V_FLAG} -o $(OUTPUT_DIR)/operator cmd/manager/main.go
+	$(Q)GOCACHE=$(GOCACHE) GOARCH=$(ARCH) GOOS=$(OS) go build ${V_FLAG} -o $(OUTPUT_DIR)/operator cmd/manager/main.go
```

Comment:
Without this, We get the following in CI:
```
failed to initialize build cache at /.cache/go-build: mkdir /.cache: permission denied
failed to initialize build cache at /.cache/go-build: mkdir /.cache: permission denied
GOARCH= GOOS= go build -v -o /go/src/github.com/redhat-developer/service-binding-operator/out/operator cmd/manager/main.go
failed to initialize build cache at /.cache/go-build: mkdir /.cache: permission denied
```

Reply:
hm, but here we also need to use vendoring, but we do not: how about that we set `GOFLAGS="-mod vendor"` instead of COCACHE? Can you please try that?

Reply:
I'll try, yes